### PR TITLE
REVERB multi-channel E2E recipe

### PR DESCRIPTION
**Pull request by Xiaofei-Wang**
- Action items
  - [x] update RESULTS once I get it 
  - [x] Add more objective metrics

### CODE REVIEW

File: egs/reverb/asr1_2ch/run.sh [hunk 1]
```diff
@@ -0,0 +1,374 @@
+#!/bin/bash
```
Comment (sw005320) [inline egs/reverb/asr1_2ch/run.sh]: It's better to generalize this recipe so that we could use any number of channels by specifying options.

**Comment by sw005320** [comment]
@sas91, please review this PR.

**Comment by Xiaofei-Wang** [comment]
Thanks, Shinji. I'll fix them.

- [ ] `asr1_2ch` -> `asr1_multich`. Then, you could specify the number of channels during training and inference by an option.
- [ ] Please use the markdown version of RESULTS. See https://github.com/espnet/espnet/blob/master/egs/chime4/asr1_multich/RESULTS.md
- [ ] It's better to add some comparisons for speech recognition and enhancement metrics (e.g., E2E enh, pipeline enh, and noisy).
- [ ] You may use a symlink file for several duplicated files in `egs/reverb/asr1/local` and `egs/reverb/asr1_multich/local`

**Comment by sw005320** [comment]
Can you add the environmental information (e.g., espnet version etc.) obtained by https://github.com/espnet/espnet/blob/master/utils/show_result.sh to `RESULTS.md`?
See https://github.com/espnet/espnet/blob/master/CONTRIBUTING.md#how-to-contribute-to-espnet and refer https://github.com/espnet/espnet/blob/master/egs/librispeech/asr1/RESULTS.md

**Comment by Xiaofei-Wang** [comment]
> Can you add the environmental information (e.g., espnet version etc.) obtained by https://github.com/espnet/espnet/blob/master/utils/show_result.sh to `RESULTS.md`?
> See https://github.com/espnet/espnet/blob/master/CONTRIBUTING.md#how-to-contribute-to-espnet and refer https://github.com/espnet/espnet/blob/master/egs/librispeech/asr1/RESULTS.md

Hi Shinji, please check if it's okay: )

File: egs/reverb/asr1_multich/run.sh [hunk 2]
```diff
@@ -0,0 +1,395 @@
+#!/bin/bash
+
+# Copyright 2018 Johns Hopkins University (Shinji Watanabe)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh
```
Comment (sw005320) [inline egs/reverb/asr1_multich/run.sh]:
```suggestion
. ./path.sh || exit 1;
```

File: egs/reverb/asr1_multich/run.sh [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+. ./cmd.sh
```
Comment (sw005320) [inline egs/reverb/asr1_multich/run.sh]:
```suggestion
. ./cmd.sh || exit 1;
```

File: egs/reverb/asr1_multich/run.sh [hunk 4]
```diff
@@ -0,0 +1,395 @@
...
+wsj0=/export/corpora5/LDC/LDC93S6B            # JHU setup
+wsj1=/export/corpora5/LDC/LDC94S13B           # JHU setup
+wavdir=${PWD}/wav # place to store WAV files
+
+# exp tag
+tag="" # tag for managing experiments.
+
+. utils/parse_options.sh || exit 1;
+
+. ./path.sh
```
Comment (sw005320) [inline egs/reverb/asr1_multich/run.sh]: Remove duplicated `. ./path.sh` and `. ./cmd.sh`.

**Comment by sas91** [comment]
> @sas91, please review this PR.

Thanks, I will check it.

**Comment by sw005320** [comment]
Thanks a lot @Xiaofei-Wang!

**Comment by sw005320** [comment]
> > @sas91, please review this PR.
> 
> Thanks, I will check it.

@sas91, I just merged this PR, but you can still continue your review and we may ask @Xiaofei-Wang to modify it with another PR.

**Pull request merged**